test/examples/ucp_hello_world.c

```diff
@@ -85,6 +85,12 @@ static size_t peer_addr_len;
 
 static ucs_status_t parse_cmd(int argc, char * const argv[], char **server_name);
 
+static void set_msg_data_len(struct msg *msg, uint64_t data_len)
+{
+    mem_type_memcpy(STRUCT_FIELD_PTR(msg, struct msg, data_len),
```

Comment:
can't we just take `&msg->data_len` ?

Reply:
taking its address would work even if it's not accessible

Reply:
> taking its address would work even if it's not accessible

fixed